src/js/builtins/ProcessObjectInternals.ts

```diff
@@ -344,3 +344,47 @@ export function setMainModule(value) {
   $putByIdDirectPrivate(this, "main", value);
   return true;
 }
+
+type InternalEnvMap = Record<string, string>;
+
+export function windowsEnv(internalEnv: InternalEnvMap, envMapList: Array<string>) {
+  // The use of String(key) here is intentional because Node.js as of v21.5.0 will throw
+  // on symbol keys as it seems they assume the user uses string keys:
+  //
+  // it throws "Cannot convert a Symbol value to a string"
+
+  return new Proxy(internalEnv, {
+    get(_, p) {
+      return typeof p === "string" ? Reflect.get(internalEnv, p.toUpperCase()) : undefined;
```

Comment:
Can you add a `TODO` to use intrinsics instead of `Reflect`? We don't need to use Reflect here at all.

Reply:
no i will just fix it